### PR TITLE
Trivial discovery updates

### PR DESCRIPTION
**Pull request by radinat**
Resolves L2B-6810

### CODE REVIEW

File: packages/backend/discovery/amarok/ethereum/diffHistory.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+
+# Diff at Tue, 20 Aug 2024 07:24:21 GMT:
+
+- author: Radina Talanova (<nt.radina@gmail.com>)
+- comparing to: main@5417c4717b5cefeed17cd8419a7eb2dda22d4206 block: 20317972
+- current block number: 20568320
+
+## Description
+
+The optimistic mode is deactivated on Ethereum by a watcher (0x56dD71fffD089EdAdbA8eCdaaDb94269713f8f4d) and "slow" mode is active. All roots must now be passed via canonical rollup AMBs. On their bridge UI they have this message: "Due to adjustments with routers and liquidity pools this month, transaction processing on several paths may be slower than usual."
```
Comment (lucadonnoh) [inline packages/backend/discovery/amarok/ethereum/diffHistory.md:11]: might be worth it adding it as discovery description

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

**Comment by lucadonnoh** [comment]
the rest lgtm

File: packages/backend/discovery/transporter/ethereum/diffHistory.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
+![image](https://github.com/user-attachments/assets/47fb0522-1b7b-4652-968f-4e0b98431feb)Generated with discovered.json: 0x42131931c0d0c0ad4a42dce8b32030af275c9cd4
```
Comment (lucadonnoh) [inline packages/backend/discovery/transporter/ethereum/diffHistory.md:1]: wrong placement

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/backend/discovery/amarok/ethereum/diffHistory.md (hunk 1 above)
Comment (radinat) [reply packages/backend/discovery/amarok/ethereum/diffHistory.md:11]: Added

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**